Symfony/CS/Tests/Fixer/Contrib/PhpUnitStrictFixerTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+class PhpUnitStrictFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideTestFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideTestFixCases()
+    {
+        $fixerReflection = new \ReflectionClass($this->getFixer());
+        $propertyReflection = $fixerReflection->getProperty('phpUnitMethods');
+        $propertyReflection->setAccessible(true);
+        $methodsMap = $propertyReflection->getValue($this->getFixer());
+
+        $cases = array(
+            array('<?php $self->foo();'),
+        );
+
+        foreach ($methodsMap as $methodBefore => $methodAfter) {
+            $cases[] = array("<?php \$sth->$methodBefore(1, 1);");
+            $cases[] = array("<?php \$sth->$methodAfter(1, 1);");
+            $cases[] = array(
+                "<?php \$this->$methodAfter(1, 2);",
+                "<?php \$this->$methodBefore(1, 2);",
+            );
+            $cases[] = array(
+                "<?php \$this->$methodAfter(1, 2); \$this->$methodAfter(1, 2);",
+                "<?php \$this->$methodBefore(1, 2); \$this->$methodBefore(1, 2);",
+            );
+            $cases[] = array(
+                "<?php \$this->$methodAfter(1, 2, 'descr');",
+                "<?php \$this->$methodBefore(1, 2, 'descr');",
+            );
+            $cases[] = array(
+                "<?php \$this->/*aaa*/$methodAfter \t /**bbb*/  ( /*ccc*/1  , 2);",
+                "<?php \$this->/*aaa*/$methodBefore \t /**bbb*/  ( /*ccc*/1  , 2);",
+            );
+        }
```

Comment:
It would be nice to test with some other types then `int`s as well.
Maybe some other scaler and a simple object? 
And a more complex one like:

```php
$this->assertEquals($expectedTokens->count() + 10, $tokens->count() ? 10 : 20 , 'Test');`
```